tests/css_less/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -3393,20 +3396,28 @@ label {
 
 // extend #7977
 .class {
-  &:extend(
-    .some-class .some-other-class .some-very-loooooooooooooong-class all
-  );
+  &:extend(.some-class
+    .some-other-class
+    .some-very-loooooooooooooong-class
+    all);
```

Comment:
Looks ugly, can we add `newline` after `(` and before `)`?

Reply:
ya, I realized it's ugly, then I found [this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBDJcAe8oBMAUAdAM4QC2cAtGADarHEA6UABGyyeVRDABZwAnanQbN2HUhUoA3QQE9KNCMpWq1KqAHNh9Jq3aoaNAJTAAvi2bN0UHkUlVause05Se-IU9H62bqrICCkrqoapaOj7ihibmllBWUIToWDgEhACWUAAOAK4wSLgZxKgARjRwuKZmIAA0IBDZMBnQxMigqAICEADuAAqdCG0ohj2ocm31pQKoYADWcDAAytmzWZrIMAK5cPW8MGQ0AOq8GfDEq2BwS0NnGdJncsjgDHUgWcSCMH0zmmSoyAAZoZPvUAFbETAAIRm80WS1QFAAMlk4ECQbsQBDMEt1hUAIq5HhopDAmigkCrASfATPCqverZARZGBHDK4PjIAAcAAZGd1PkcZtlnky4DTZG8AI5E+A-RrDNDEShQOCVSpvARwGUZLU-VB-AGkjH1T5kDKbbaY4h4uCE4no8mYmBlNkc3jIABM9S2qAyNHWAGFyP9nuKAKxvXKfAAqZWGZIp0h2AEk8AhlmBmU0AIJ4JYwOQVR2fMxmIA), what do we do?

Reply:
Output it as selectors, because it is selectors if they not fit in one line ([example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA6AZhCAaABKgIwEMAnPQogL3IBst6HGnmGoBzXYAHSl1x4C+IbCAgAHGAEtoAZ2ShSJCAHcACqQRyURGsqIBPOSIIkiYANZwYAZTFnJ7ZDBIBXOCIAWMALY0A6h6S8DJ2YHDWmkGSAG5B+sjgMkYgDjJwJDCqpmzeRMjoOmkiAFYyAB4AQqYWVtZE3nAAMg5w+YXuIKVl1g5sNHAAii4Q8G00RSB2JGkkCf1JwpMkDjB+kgAmMB7IABwADCJiSml+pmIJR3Az0a0iAI7D8FniWiBEMgC0UHBw67+LJDgD0kgKyRByeSQBXGHTS3kkTlcsN6-SGI1aUPaIhgRAIa022yQACZsaZJDRegBhCDeXIJK4AVkWLjSABVcVpoRNom4AJJQP6waxgZYSACCAusMH0-TGaQEAiAA)) :smile: 

```lesss
&:extend(
  .some-class
  .some-other-class
  .some-very-loooooooooooooong-class
  all
);
```

Reply:
Sorry, but  my link  is Stable version.

Reply:
We should fix it

Reply:
This can  get  really  complicated, like  chain and BinaryExpression in js

```css
a:nth-last-child(2n + 1)
  :not(
    .some-class
    .some-other-class
    .some-very-loooooooooooooooooong-class
    all
  )
~ a:nth-last-child(2n + 1)
  :not(
    .some-class
    .some-other-class
    .some-very-loooooooooooooooooong-class
    all
    + a:nth-last-child(2n + 1)
      :not(
        .some-class
        .some-other-class
        .some-very-loooooooooooooooooong-class
        all
      )
  )
::before {
}
```

Reply:
`:not` can't be used `:not`. See https://developer.mozilla.org/en-US/docs/Web/CSS/:not.

Reply:
Thanks, still there could be pseudo-class inside pseudo-class. I changed example to `:has`

Reply:
Let's improve it in #8178